src/styles/theme/themes/light.ts

```diff
@@ -137,7 +137,7 @@ const lightTheme = {
         },
         [SCREENS.SETTINGS.TROUBLESHOOT]: {
             backgroundColor: colors.blue700,
```

Comment:
Should this be changed too? We don't use a color behind the status bar anymore on this page.

Reply:
I also think we should change it to alight with other settings page and use `colors.productLight100`
![Simulator Screenshot - iPhone 15 Plus - 2024-06-24 at 01 34 43](https://github.com/Expensify/App/assets/59907218/a9deb0e0-56d4-433e-abd0-84b6a6c1ae44)

Reply:
Works for me